soh/src/overlays/gamestates/ovl_file_choose/z_file_choose.c

```diff
@@ -298,6 +298,7 @@ void FileChoose_UpdateMainMenu(GameState* thisx) {
         Randomizer_LoadItemLocations(fileLoc, silent);
         Randomizer_LoadMerchantMessages(fileLoc);
         Randomizer_LoadMasterQuestDungeons(fileLoc);
+        Randomizer_LoadEntranceOverrides(fileLoc, silent);
```

Comment:
Do we not need a `Randomizer_LoadEntranceOverrides("");` near the top of `FileChoose_LoadGame`?

Reply:
Since as its coded now, the entrance overrides are pulled from gSaveContext during sram_opensave, the load function is a noop here. But I probably should add it here for posterity sake and to work with your fast file select changes (ER works fine as is with FFS since gSaveContext isn't augmented)